google-cloud-clients/google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/v1/MessageDispatcherTest.java

```diff
@@ -92,6 +93,7 @@ public void sendAckOperations(
                 sentModAcks.add(ModAckItem.of(ackId, modack.deadlineExtensionSeconds));
               }
             }
+            return null;
```

Comment:
Instead of null can we send `ApiFutures.immediateFuture(null)`.. By doing this you won't result in NPE